cmd/paratime/list.go

```diff
@@ -48,3 +52,19 @@ var listCmd = &cobra.Command{
 		table.Render()
 	},
 }
+
+func formatDenominations(denominations map[string]*config.DenominationInfo) string {
+	fmtDenomArray := []string{}
+	fmtDenom := ""
+	for nativeKey, denom := range denominations {
+		// NativeDenominationKey = "_"
+		if nativeKey == "_" {
+			fmtDenom = fmt.Sprintf("%s[%d] (*)", denom.Symbol, denom.Decimals)
+		} else {
+			fmtDenom = fmt.Sprintf("%s[%d]", denom.Symbol, denom.Decimals)
+		}
+
+		fmtDenomArray = append(fmtDenomArray, fmtDenom)
+	}
+	return (strings.Join(fmtDenomArray, ", "))
```

Comment:
Are newlines supported by the table formatter? Maybe that would look better.

Reply:
I think there is a problem with tests. On my computer, there is one space between the `NETWORK` and `PARATIME` columns, but in tests, there are multiple spaces between those two columns:

```diff --git a/examples/paratime/00-list.out b/examples/paratime/00-list.out
index fd93a16..0645e72 100644
--- a/examples/paratime/00-list.out
+++ b/examples/paratime/00-list.out
@@ -1,9 +1,9 @@
-NETWORK PARATIME        ID                                                                      DENOMINATION(S) 
-mainnet cipher          000000000000000000000000000000000000000000000000e199119c992377cb        ROSE[9] (*)    
-mainnet emerald         000000000000000000000000000000000000000000000000e2eaa99fc008f87f        ROSE[18] (*)   
-mainnet sapphire (*)    000000000000000000000000000000000000000000000000f80306c9858e7279        ROSE[18] (*)   
-testnet cipher          0000000000000000000000000000000000000000000000000000000000000000        TEST[9] (*)    
-testnet emerald         00000000000000000000000000000000000000000000000072c8[215](https://github.com/oasisprotocol/cli/actions/runs/9076794590/job/24940286293?pr=237#step:7:216)e60d5bca7        TEST[18] (*)   
-testnet pontusx         0000000000000000000000000000000000000000000000004febe52eb412b421        TEST[18]       
-                                                                                                EUROe[18] (*)  
-testnet sapphire (*)    000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c        TEST[18] (*)
+NETWORK	PARATIME    	ID                                                              	DENOMINATION(S) 
+mainnet	cipher      	000000000000000000000000000000000000000000000000e199119c992377cb	ROSE[9] (*)    	
+mainnet	emerald     	000000000000000000000000000000000000000000000000e2eaa99fc008f87f	ROSE[18] (*)   	
+mainnet	sapphire (*)	000000000000000000000000000000000000000000000000f80306c9858e7279	ROSE[18] (*)   	
+testnet	cipher      	0000000000000000000000000000000000000000000000000000000000000000	TEST[9] (*)    	
+testnet	emerald     	00000000000000000000000000000000000000000000000072c8215e60d5bca7	TEST[18] (*)   	
+testnet	pontusx     	0000000000000000000000000000000000000000000000004febe52eb412b421	TEST[18]       	
+       	            	                                                                	EUROe[18] (*)```

Reply:
Never mind, I solved it. :)